modules/openapi-generator/src/main/resources/dart/libraries/dio/build.yaml.mustache

```diff
@@ -0,0 +1,18 @@
+targets:
```

Comment:
I would move this file under json_serializer if possible, let's keep things separated to avoid guessing to which serialization it belong